### PR TITLE
Bike list

### PR DESCRIPTION
**Pull request by tmassalski**

### CODE REVIEW

File: src/api/BikelyApi.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export class BikelyApi {
-  static apiUrl = 'https://coderscamp-bikely.herokuapp.com/';
+  static apiUrl = 'http://localhost:8080';
+  // static apiUrl = 'https://coderscamp-bikely.herokuapp.com/';
```
Comment (MarKust71) [inline src/api/BikelyApi.js]: Unnecessary comment, I think, but I know the purpose. By the way, leaving the slash at the end of the Uri/Url brings some troubles. It depends, of course, on how the particular controller has been configured and we need, maybe, some unified approach where this slash should be provided when needed.

File: src/components/reservations/BikeTable.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+          count={rows.length}
+          rowsPerPage={rowsPerPage}
+          page={page}
+          onPageChange={handleChangePage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
+        />
+      </Paper>
+    </div>
+  );
+};
```
Comment (MarKust71) [inline src/components/reservations/BikeTable.js:238]: 239 lines for the component file it must be a mistake ;) but if it worked in this form let Mikołaj do refactor of this and then you will get an example he to work with such things in the future. ok?

File: src/components/reservations/Filters.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      </MuiPickersUtilsProvider>
+      <FormControl error={inputErrors.selectedBikes}>
+        <Button variant="contained" color="primary" onClick={handleClick}>
+          Create Reservation
+        </Button>
+        {inputErrors.selectedBikes && <FormHelperText>Select bike</FormHelperText>}
+      </FormControl>
+    </Box>
+  );
+};
```
Comment (MarKust71) [inline src/components/reservations/Filters.js:203]: Same as above - too many things in one file. But focus on the logic, let Mikołaj do the refactor.

File: src/components/reservations/index.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+            </LocalizationProvider>
+          </Paper>
+        </Grid>
+        <Grid item xs={9}>
+          <BikeTable bikes={filterTable(bikes)} onBikeSelection={handleBikeSelection} />
+        </Grid>
+      </Grid>
+    </Container>
+  );
+};
```
Comment (MarKust71) [inline src/components/reservations/index.js:141]: again...

**Review by MarKust71 — DISMISSED** [review]

File: src/api/BikelyApi.js (hunk 1 above)
Comment (tmassalski) [reply src/api/BikelyApi.js]: Ok, comment and slash removed.

File: src/components/reservations/BikeTable.js (hunk 2 above)
Comment (tmassalski) [reply src/components/reservations/BikeTable.js:238]: ok

File: src/components/reservations/Filters.js (hunk 3 above)
Comment (tmassalski) [reply src/components/reservations/Filters.js:203]: ok

File: src/components/reservations/index.js (hunk 4 above)
Comment (tmassalski) [reply src/components/reservations/index.js:141]: ok

**Review by MarKust71 — APPROVED** [review]

**Pull request merged**